timeline.js

```diff
@@ -0,0 +1,43 @@
+import { ActivityPubDB } from "./db.js";
+
+class ReaderTimeline extends HTMLElement {
+  constructor() {
+    super();
+    // Default outbox URLs to fetch and display in the timeline
+    this.outboxUrls = [
+      "ipns://staticpub.mauve.moe/outbox.jsonld",
+      "ipns://hypha.coop/outbox.jsonld",
+    ];
+  }
+
+  connectedCallback() {
+    this.initTimeline();
+  }
+
+  async initTimeline() {
+    const db = await ActivityPubDB.load();
+
+    // Clear existing content
+    this.innerHTML = "";
+
+    for (const outboxUrl of this.outboxUrls) {
+      try {
+        console.log("Ingesting actor from outbox URL:", outboxUrl);
+        await db.ingestActor(outboxUrl);
+
+        const outboxElement = document.createElement("distributed-outbox");
```

Comment:
we shouldn't be outputting several outboxes. we need to instead have the timeline use `db.searchNotes` and render note elements.

From there we could filter by author or time ranges or tags.